nfc/NfcCx/Cx/NfcCxRF.cpp

```diff
@@ -141,6 +141,33 @@ NfcCxRFInterfaceGetSEEvent(
 }
 
 VOID FORCEINLINE
+NfcCxRFInterfaceClearTagRemoteDevList(
+    _Inout_ PNFCCX_RF_INTERFACE RFInterface
+)
+{
+    uint8_t bIndex = 0x00;
+
+    if (RFInterface->pLibNfcContext->pRemDevList != NULL) 
+    {
+        for (bIndex = 0; bIndex < RFInterface->pLibNfcContext->uNoRemoteDevices; bIndex++)
+        {
+            if (RFInterface->pLibNfcContext->pRemDevList[bIndex].psRemoteDevInfo != NULL)
+            {
+                if ((RFInterface->pLibNfcContext->pRemDevList[bIndex].psRemoteDevInfo->RemDevType == phLibNfc_eISO14443_A_PCD) ||
+                    (RFInterface->pLibNfcContext->pRemDevList[bIndex].psRemoteDevInfo->RemDevType == phLibNfc_eISO14443_B_PCD))
+                {
+                    RFInterface->pLibNfcContext->bIsHCEConnected = FALSE;
```

Comment:
HCE  RemDev instances would be of specified A_PC B_PCD types.
In review reworked code,
we have removed NfcCxRFInterfaceClearTagRemoteDevList function, And added uNoRemoteDevices > 1 case ( multiple RemDev) in existing NfcCxRFInterfaceClearRemoteDevList to avoid redundancy.